### ISSUE TITLE
Where can I find the logits of DINO? 

### ISSUE THREAD

**Issue by Liuziyu77** [open-mmlab/mmdetection#11301]
Where can I find the logits of DINO? I am using the Dino with Swin, and inference with DetInferencer.

Here is my code:
```
inferencer = DetInferencer(model='path to my model', weights='path to my model')
result = inferencer('demo/demo.jpg',  out_dir='outputs/', no_save_pred=False)
```
